src/shared/lib/newsletterConsentsPageLocalisation.ts

```diff
@@ -0,0 +1,84 @@
+import {
+  GeoLocation,
+  PermissionedGeolocation,
+} from '@/shared/model/Geolocation';
+import { Newsletters } from '@/shared/model/Newsletter';
+import { CONSENTS_NEWSLETTERS_PAGE } from '../model/Consent';
+
+// Permissions a geolocation if cmp consent is true
+export const getPermissionedGeolocation = (
+  cmpConsentState: boolean | undefined,
+  geolocation: GeoLocation | undefined,
+): GeoLocation | PermissionedGeolocation | undefined =>
+  !!cmpConsentState && geolocation === 'AU' ? 'AU_permissioned' : geolocation;
```

Comment:
Just finding this logic slightly hard to read, I wonder if this could be improved somehow?

It might be slightly longer, but an if statement is easier to read, that's just my opinion though

```suggestion
export const getPermissionedGeolocation = (
  cmpConsentState: boolean | undefined,
  geolocation: GeoLocation | undefined,
): GeoLocation | PermissionedGeolocation | undefined => {
  if (!!cmpConsentState && geolocation === 'AU') {
    return 'AU_permissioned'
  }
  return geolocation;
}
```

Reply:
👍 yes, will amend